Tests/SourceKitLSPTests/FoldingRangeTests.swift

```diff
@@ -357,4 +357,52 @@ final class FoldingRangeTests: XCTestCase {
       expectedRanges: [FoldingRangeSpec(from: "1️⃣", to: "2️⃣")]
     )
   }
+
+  func testFoldArgumentsForConditionalIfCompileDirectives() async throws {
+    try await assertFoldingRanges(
+      markedSource: """
+        1️⃣#if DEBUG
+            let foo = "x"
+        2️⃣#endif
+        """,
+      expectedRanges: [
+        FoldingRangeSpec(from: "1️⃣", to: "2️⃣")
+      ]
+    )
+  }
+
+  func testFoldArgumentsForConditionalElseIfCompileDirectives() async throws {
+    try await assertFoldingRanges(
+      markedSource: """
+        1️⃣#if DEBUG
+          let foo = "x"
+        2️⃣3️⃣#elseif TEST
```

Comment:
There’s not really a reason to specify two markers here. You could just use `2` for both the first range’s end and the next range’s start.